packages/pluggableWidgets/gallery-text-filter-native/src/components/FilterComponent.tsx

```diff
@@ -59,6 +68,20 @@ export default function FilterComponent(props: FilterComponentProps): ReactEleme
         }
     }, [inputRef]);
 
+    const onFocus = (): void =>
```

Comment:
can we use `isFocused` instead of adding a new state?
you may use something like
`textInputContainerStyle = {
            ...props.styles?.textInputContainer,
            ...{inputRef.isFocused() ? props.styles?.textInputContainerOnFocus: undefined}
        }`